sdk/storage/test/test_base.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "test_base.hpp"
+
+namespace Azure  {  namespace Storage  {  namespace Test  {
+
+constexpr const char* TestUtility::k_STANDARD_STORAGE_CONNECTION_STRING;
+constexpr const char* TestUtility::k_PREMIUM_STORAGE_CONNECTION_STRING;
+constexpr const char* TestUtility::k_BLOB_STORAGE_CONNECTION_STRING;
+constexpr const char* TestUtility::k_PREMIUM_FILE_CONNECTION_STRING;
+constexpr const char* TestUtility::k_ADLS_GEN2_CONNECTION_STRING;
+constexpr const char* TestUtility::k_ACCOUNT_SAS;
```

Comment:
We already have definition in .hpp file, do we still need these in .cpp file?

Reply:
Yes, otherwise some compiler would report compiler issue on `k_STANDARD_STORAGE_CONNECTION_STRING` not being defined.